code/modules/client/preference/preferences.dm

```diff
@@ -68,6 +68,10 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 	var/clientfps = 63
 	var/atklog = ATKLOG_ALL
 	var/fuid							// forum userid
```

Comment:
unrelated but why is this indented 7 times

Reply:
Will decrease down to 4. Just because I can and other comments are at 4 tabs